index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8"/>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet'>
+  <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>
+  <link rel="stylesheet" href="css/mobile-version.css"/>
+  <link rel="stylesheet" href="css/desktop-version.css"/>
+  <script src="https://kit.fontawesome.com/8c5a2b9118.js" crossorigin="anonymous"></script>
+  <title>Youcode</title>
+</head>
+<body>
+  <!-- Main page section -->
+  <section id="main-page">
+    <header>
+      <div class="social-nav">
+        <ul>
+          <li><i class="fa fa-brands fa-facebook-f"></i></li>
+          <li><i class="fa fa-brands fa-twitter"></i></li>
+          <li>English</li>
+          <li>My Page</li>
+          <li>Logout</li>
+        </ul>
+      </div>
+      <div class="navbar">
+        <a class="my-logo" href="#headline">YouCode</a>
+        <ul class="menu">
+          <li><a class="menuItem" href="about.html">About</a></li>
+          <li><a class="menuItem" href="#" style="color: lightgray">Program</a></li>
+          <li><a class="menuItem" href="#" style="color: lightgray">Join</a></li>
+          <li><a class="menuItem" href="#" style="color: lightgray">Sponsor</a></li>
+          <li><a class="menuItem" href="#" style="color: lightgray">News</a></li>
+        </ul>
+        <ul class="desktop-menu">
+          <li><a class="" href="about.html">About</a></li>
+          <li><a class="" href="#"style="color: lightgray">Program</a></li>
+          <li><a class="" href="#" style="color: lightgray">Join</a></li>
+          <li><a class="" href="#" style="color: lightgray">Sponsor</a></li>
+          <li><a class="" href="#" style="color: lightgray">News</a></li>
+          <button class="navbar-btn">CC Campaign</button>
+         </ul>
```

Comment:
- [ ] For best practices and clean code, I think it would be better if you remove the inline styling on all your navigation options(`style="color: lightgray"`) and ensure that your header stick as much as possible to the template design(e.g.: text color, spacing, alignment,...).
### Template Design
![Screenshot from 2023-02-14 01-01-55](https://user-images.githubusercontent.com/83042742/218594506-808eb816-5f22-4aa1-ad29-cbebf115eb29.png)

### Your Design
![Screenshot from 2023-02-14 01-02-35](https://user-images.githubusercontent.com/83042742/218594541-8b7c6a95-145d-46f1-8df3-f7844ab2a06e.png)